infrastructure/azure-pipelines-dev.yml

```diff
@@ -45,9 +45,6 @@ jobs:
       - script: echo ##vso[task.setvariable variable=NPM_CONFIG_USERCONFIG]$(npm_config_user_config)
         displayName: "Set Custom .npmrc Location"
 
-      - script: CYPRESS_INSTALL_BINARY=/opt/cypress/13.10.0/cypress.zip pnpm install cypress
```

Comment:
@brindasasi I'm not sure if there is anything manual that needs to be changed or removed in Azure?

Reply:
This is the only place needs to be changed. Let me trigger the dev pipeline with this change